DOCS.md

```diff
@@ -376,15 +376,23 @@ If `type` is `"event"` then the object will also have those fields:
 - `author`: The person who performed the event.
 
 If `type` is `"typ"` then the object will have the following fields:
-- `isTyping`: Boolean representing whether or not a person started typing
-- `from`: ID of the user who started/stopped typing
-- `threadID`: Current threadID
-- `from_mobile`: Boolean representing whether or not the person's using a mobile device to type
-
-If `type` is `"read_receipt"` then the object will have the following fileds:
-- `reader`: ID of the user who just read the message
-- `time`: the time at which the reader read the message
-- `threadID`: the thread in which the message was read
+- `isTyping`: Boolean representing whether or not a person started typing.
+- `from`: ID of the user who started/stopped typing.
+- `threadID`: Current threadID.
+- `from_mobile`: Boolean representing whether or not the person's using a mobile device to type.
```

Comment:
We just updated `from_mobile` to `fromMobile`, can you make the same change so we can merge :)